app/src/main/res/values/strings.xml

```diff
@@ -186,6 +186,7 @@
     <string name="settings_dark_palette">Dark Palette</string>
     <string name="settings_switch_automatically">Switch Automatically</string>
     <string name="settings_brightness">Brightness</string>
+    <string name="settings_number_row">Number/Row</string>
```

Comment:
ah, you mean number_per_row. settings_photos_per_row might be the more intuitive name.

Reply:
Ok